spec/models/identity_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Identity do
+  let(:identity) { build(:identity) }
+  let(:no_name) { build(:identity, first_name: "") }
+
+  describe "validation" do
+
+    context "without required attributes" do
+      it "should be invalid" do
```

Comment:
Use of `should` is generally discouraged http://www.betterspecs.org/#should

Reply:
old habits...